sdk/identity/azure-identity/azure/identity/_credentials/default.py

```diff
@@ -42,3 +44,9 @@ def __init__(self, **kwargs):
             )
 
         super(DefaultAzureCredential, self).__init__(*credentials)
+
+    def get_token(self, *scopes, **kwargs):  # pylint:disable=unused-argument
+        try:
+            return super(DefaultAzureCredential, self).get_token(*scopes, **kwargs)
+        except ClientAuthenticationError as e:
+            raise ClientAuthenticationError(message="{}\nPlease visit the python azure identity sdk docs on github to learn what options defaultAzureCredential supports".format(e.message))
```

Comment:
@kaerm do we already have an aka link to https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/identity/azure-identity?

Reply:
how about https://aka.ms/sdk-identity ?

Reply:
Must distinguish languages; python-sdk-identity?

Reply:
you are correct - long day: https://aka.ms/python-sdk-identity

Reply:
Thanks folks! (for both the aka link and sanity checking that we're OK putting URIs in exceptions) Adjusting.  :)